lib/linux/audio.nit

```diff
@@ -56,7 +56,12 @@ redef class Sound
 		self.native = native
 	end
 
-	redef fun play
+	redef fun play do play_channel(-1, 0)
+
+	# Play this sound on `channel` (or any channel if -1) and return the channel
+	#
+	# Repeat the sound `repeats` times.
+	fun play_channel(channel, repeats: Int): Int
```

Comment:
why change the vocabulary? it was `loops` in `Mix::play_channel` it is now `repeats` here